src/pathfinding_service/model/token_network.py

```diff
@@ -155,10 +155,16 @@ def _get_address_metadata(
                     UserPresence.ONLINE,
                     UserPresence.UNAVAILABLE,
                 ]:
+                    displayname = (
+                        self.reachability_state._displayname_cache.userid_to_displayname.get(
```

Comment:
doesn't mypy complain that you access an unknown field from `AddressReachabilityProtocol` ?

Reply:
No, MyPy is somehow fine with this line.